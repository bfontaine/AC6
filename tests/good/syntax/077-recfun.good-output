Line 1, characters 18-19: Error (Parsing)
  Unknown error.

