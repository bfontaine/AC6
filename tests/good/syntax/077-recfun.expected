def f (x : { U }) =
  g (U)
with g (x : { U }) =
  f (U)
with h (x : { U }) (y : { U }) =
  g x
